core/App/navigators/ContactStack.tsx

```diff
@@ -12,12 +11,11 @@ import { createDefaultStackOptions } from './defaultStackOptions'
 
 const ContactStack: React.FC = () => {
   const Stack = createStackNavigator<ContactStackParams>()
-  const { t } = useTranslation()
   const theme = useTheme()
   const defaultStackOptions = createDefaultStackOptions(theme)
   return (
     <Stack.Navigator screenOptions={{ ...defaultStackOptions }}>
-      <Stack.Screen name={Screens.Contacts} component={ListContacts} options={{ headerBackTitle: t('Global.Back') }} />
+      <Stack.Screen name={Screens.Contacts} component={ListContacts} options={{ headerBackTitle: ' ' }} />
```

Comment:
Might be better to use the `headerBackTitleVisible` option here and remove the `headerBackTitle` option altogether.

Reply:
Okay I changed it to `headerBackTitleVisible: false`